sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -91,6 +134,53 @@ export default {
     const studentsLastName = ref("");
     const studentsDateOfBirth = ref();
     const initialData = ref();
+    const formioUtils = useFormioUtils();
+    const isFirstPage = ref(true);
+    const isLastPage = ref(false);
+    const showNav = ref(false);
+    let formInstance: any;
+
+    const wizardSubmit = () => {
+      formInstance.submit();
+    };
+
+    const formLoaded = async (form: any) => {
+      showNav.value = true;
+      // Emit formLoadedCallback event to the parent, so that parent can
+      // perform the parent specific logic inside parent on
+      // form is loaded
+      context.emit("formLoadedCallback", form);
```

Comment:
I guess that this was needed for the student application but I do not see a reason to have it here. I would suggest removing it.

Reply:
That's right. Thanks!

Reply:
Was it changed, I am not able to see the file changed.

Reply:
Yes, please check again.